src/test/kotlin/io/inventi/eventstore/eventhandler/CatchupSubscriptionsMeterRegistryIntegrationTest.kt

```diff
@@ -0,0 +1,94 @@
+package io.inventi.eventstore.eventhandler
+
+import com.github.msemys.esjc.EventStore
+import io.inventi.eventstore.EventStoreIntegrationTest
+import io.inventi.eventstore.EventStoreToolsSubscriptionsConfiguration
+import io.inventi.eventstore.eventhandler.annotation.EventHandler
+import io.inventi.eventstore.eventhandler.events.EventType
+import io.inventi.eventstore.eventhandler.model.EventIds
+import io.inventi.eventstore.eventhandler.util.DataBuilder
+import io.inventi.eventstore.eventhandler.util.DataBuilder.event
+import io.inventi.eventstore.eventhandler.util.DataBuilder.eventId
+import io.inventi.eventstore.eventhandler.util.Handler
+import io.inventi.eventstore.eventhandler.util.WithEventstoreOperations
+import io.inventi.eventstore.eventhandler.util.WithMeterRegistryAssertions
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Import
+import org.springframework.test.annotation.DirtiesContext
+import org.springframework.test.context.ActiveProfiles
+
+@SpringBootTest(classes = [EventStoreToolsSubscriptionsConfiguration::class])
+@Import(CatchupSubscriptionsMeterRegistryIntegrationTest.Config::class)
+@ActiveProfiles("test")
+@DirtiesContext // ensures that eventStore bean is not reused, because each bean has a different test container port
+class CatchupSubscriptionsMeterRegistryIntegrationTest : EventStoreIntegrationTest(), WithMeterRegistryAssertions, WithEventstoreOperations {
+
+    @Autowired
+    override lateinit var eventStore: EventStore
+
+    @Autowired
+    override lateinit var meterRegistry: MeterRegistry
+
+    @Autowired
+    private lateinit var handler: Handler
+
+    @Test
+    fun `when handler is subscribed to stream it's gauge value is seen as active`() {
```

Comment:
this naming does not follow our usual conventions, e.g. `shows metric's gauge as active when handler is subscribed to stream`